plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -96,11 +96,40 @@
         <mavenCoordinates artifactId="jstl" groupId="javax.servlet" version="1.2" />
       </libraryFile>
     </library>
+    
+    <library
+          id="googleapiclient"
+          group="clientapis"
+          name="Google API Client Library for Java"
+          tooltip="%googleapiclient.tooltip"
+          siteUri="https://developers.google.com/api-client-library/java/" >
+      <libraryFile
+            javadocUri="https://google.github.io/google-api-java-client/releases/1.21.0/javadoc/index.html">
```

Comment:
these javadocs for the new library files are pointing to 1.21.0 while the maven version is 1.22.0.